### ISSUE TITLE
Support for Hudl

### ISSUE THREAD

**Issue by jtv7** [rg3/youtube-dl#6562]
Would it be possible to support http://www.hudl.com ?  It uses JSON for the video URLs, so hopefully it won't be too difficult :)

**Comment by yan12125** [comment]
[Please provide an URL](https://github.com/rg3/youtube-dl#is-the-description-of-the-issue-itself-sufficient).

**Comment by KeenRivals** [comment]
Here is an example URL.
http://www.hudl.com/athlete/2538180/highlights/149298443/v2

**Comment by TRox1972** [comment]
As of version 2016.05.21.2, the generic extractor is able to download the provided link.

**Comment by yan12125** [comment]
The latest version returns two identical videos. This will be fixed in the next version.

**Comment by juliussss** [comment]
Looks like they have changed their URL scheme... (as of Aug 16, 2017)

**Old:** http://www.hudl.com/athlete/2538180/highlights/149298443/v2
**New:** http://www.hudl.com/video/3/2538180/5721b8f70428ae20e4abe221

Any chance this could be updated in latest version? Would be much appreciated x

**Comment by yan12125** [comment]
Yep Hudl now uses it's own specific logic to load videos and dedicated hudl.py is necessary.
